readthedocs/oauth/migrations/0012_add_fields_to_remote_relation_model.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.2.16 on 2020-10-05 06:13
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('socialaccount', '0003_extra_data_default_dict'),
+        ('oauth', '0011_add_remote_relation_model'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='remoterelation',
+            name='account',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='remote_relations', to='socialaccount.SocialAccount', verbose_name='Connected account'),
+        ),
+        migrations.AddField(
+            model_name='remoterelation',
+            name='active',
+            field=models.BooleanField(default=False, verbose_name='Active'),
+        ),
```

Comment:
I'm not sure what `active` means :smile:. Do you? Why do you think it's a field from the `RemoteRelation` and not for the `RemoteRepository`? I think it's not even used (it's the first time I see it), but I'm not sure.

Reply:
I have not seen it reference anywhere, Maybe we could just remove it. I assumed that it was used to make the repository unimportable by the user or something.

Reply:
I think we can just avoid it from the migration and remove it.